.travis.yml

```diff
@@ -13,6 +13,7 @@ before_script:
   - export DJANGO_SECRET_KEY=foobarbaz
 script:
   - cd vms
+  - flake8 --show-source
```

Comment:
`flake8 .` would do. We can only add it once we also fix the flake8 issues with the code.

Reply:
So should I change it or leave it for later?

Reply:
We can merge the `flake8 .` change once we also fix the issues that comes when you run this command in your local on vms. I hope this helps.

Reply:
Okay. Got it. Thanks.